### PR TITLE
Lolo's Homework

### PR DESCRIPTION
**Pull request by rienlondon**
Hey, I wondered if I have done this correctly :)Lgcash

### CODE REVIEW

**Comment by yeeyang** [comment]
I received the pull request but there are improvements that can be made :+1: Give it another go over to make sure all your tags are closed and are being properly used.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Homework 1</title>
+</head>
```
Comment (yeeyang) [inline index.html]: I'm glad you caught that closing `</head>` tag. :+1:

**Comment by yeeyang** [comment]
Well Done. Code is looking great! :100:

**Comment by rienlondon** [comment]
Thanks!

**Pull request closed without merging**